addons/binding/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/protocol/ZoneAvailableInputs.java

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.yamahareceiver.internal.protocol;
+
+/**
+ * Implement this to
+ *
+ * @author David Graeff - Initial contribution
+ */
+
+public interface ZoneAvailableInputs extends IStateUpdateable {
```

Comment:
My question was more based on why not marking it as `IStateUpdateable`, would it not be sufficient instead of having an interface `ZoneAvailableInputs` which adds nothing on top of `IStateUpdateable`